spring-cloud-starter-dataflow-server-local/src/test/resources/org/springframework/cloud/dataflow/server/local/security/oauthConfig.yml

```diff
@@ -1,6 +1,3 @@
-management:
-  security:
-    enabled: true
```

Comment:
If `management.security.enabled=true` you'll get an error from Boot saying:
```
"Access is denied. User must have one of the these roles: ADMIN"
```
when accessing `/management` endpoints